file.js

```diff
@@ -14,3 +17,84 @@ document.querySelectorAll('.link').forEach((n) => n.addEventListener('click', ()
   navbar.classList.remove('active');
   container.classList.remove('active');
 }));
+// popup window
+const myProjects = [
+  {
+    name: 'Tonic',
+    description: "Lorem Ipsum is simply dummy text of the printi my text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of typeng and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    featuredImage: 'images/SnapshootPortfolio.png',
+    tech: ['Codekit', 'GitHub', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+    liveLink: 'https://suzi216.github.io/Portofolio/',
+    sourceLink: 'https://github.com/Suzi216/Portofolio/',
+  },
+  {
```

Comment:
- [ ] kindly ensure that the mobile version is full height. your see live and see source of the mobile version are not really visible. also kindly ensure that your portfolio cant be control when the popup comes up
your version
![Screenshot (261)_LI](https://user-images.githubusercontent.com/65298048/180024431-dc9771f7-7844-4cee-ac79-52d9b8f87d26.jpg)

figma version
![Screenshot (262)](https://user-images.githubusercontent.com/65298048/180024557-aa857aab-16fe-4938-9722-eb697046fcd9.png)